project/build.properties

```diff
@@ -1 +0,0 @@
-sbt.version=0.13.7
```

Comment:
Wait, why was this removed?  Having a standard `build.properties` is pretty useful.